src/main/java/edu/hm/hafner/analysis/parser/JavacParser.java

```diff
@@ -29,13 +29,17 @@ public class JavacParser extends AbstractMavenLogParser {
                                               // whitespace followed by whitespace. This can be used for timestamps.
             + "(?:(?:\\[(WARNING|ERROR)\\]|w:|e:)\\s+)" // optional [WARNING] or [ERROR] or w: or e:
             + "([^\\[\\(]*):\\s*"             // group 1: filename
+            + "("                             // group 2: enclose matching group
+            + "("                             // enclose java and kotlin < 1.8 style
             + "[\\[\\(]"                      // [ or (
-            + "(\\d+)[.,;]*"                  // group 2: line number
-            + "\\s?(\\d+)?"                   // group 3: optional column
+            + "(\\d+)[.,;]*"                  // line number
```

Comment:
Wouldn't a change of the original 
```
[\\[\\(]?(\\d+)[.,;]\\s?(\\d+)?[\\]\\)]?\\s*
```
work out of the box?

So either ( or [ or nothing?

Reply:
that was my first approach but tests were failing as too much warnings are found.
the current solution has also failing tests because of the groups